packages/core/src/agent/__tests__/Agent.test.ts

```diff
@@ -263,10 +259,11 @@ describe('Agent', () => {
         'https://didcomm.org/messagepickup/2.0',
         'https://didcomm.org/out-of-band/1.1',
         'https://didcomm.org/present-proof/1.0',
+        'https://didcomm.org/present-proof/2.0',
```

Comment:
PPV2 was already present, right? So we were missing the feature registration before?

Reply:
Yes